README.md

```diff
@@ -35,7 +35,8 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+var Andy = Instructor.findOne({name: "Andy"})
+Andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold"})
```

Comment:
Need to...

1. Wrap the second line in a callback or promise
2. Make sure to save `Andy` after pushing the wishlistitem